internal/driver/config.go

```diff
@@ -68,7 +68,7 @@ func defaultConfig() config {
 		Trim:         true,
 		DivideBy:     1.0,
 		Sort:         "flat",
-		Granularity:  "functions",
```

Comment:
One potentially unintended consequence of this change is that the default is now unclear in the output of the "o" command in the interactive shell.

Before:
```
$ pprof profile.pb.gz
...
(pprof) o
...
  granularity               = functions            //: [addresses | filefunctions | files | functions | lines]
...
```

With the change:
```
$ go run pprof.go profile.pb.gz
...
(pprof) o
...
  granularity               =                      //: [addresses | filefunctions | files | functions | lines]
...
```

Reply:
Maybe this is reasonable given that the change proposes having different default granularities for different views.

Reply:
Or maybe we should at least show a string like "default" here? Showing empty string just looks a bit odd.

Reply:
We now show:

```
  granularity               = (default)            //: [addresses | filefunctions | files | functions | lines]
```